Kvent/views.py

```diff
@@ -52,19 +53,22 @@ def detail(request, event_id):
 
 def create_event(request):
     """ User creates the event """
-    form = EventForm(request.POST)
+    form = EventForm(request.POST, request.FILES)
     if request.method == 'POST' :
         if form.is_valid() :
+            photo = form.cleaned_data.get('photo') 
+            print(photo)
             event_name = form.data.get('event_name')
+            print(event_name)
```

Comment:
Remove _print(event_name)_

Reply:
I print to check the output data, I will delete these out.